addons/l10n_es_edi_facturae/tests/test_edi_xml.py

```diff
@@ -16,7 +16,13 @@
 
 _logger = logging.getLogger(__name__)
 
+
+def _compute_is_valid(self):
+    for cert in self:
+        cert.is_valid = True
```

Comment:
Is this patched so that the certificates would always be valid without having to keep updating them? If so, maybe a comment to explain that would be nice? :)